packages/splitio-adapter/modules/adapter/adapter.js

```diff
@@ -54,12 +54,13 @@ const ensureUser = user => ({
   ...user,
 });
 
-const initializeClient = (authorizationKey, user) => {
+const initializeClient = (authorizationKey, user, options) => {
   const factory = splitio({
     core: {
       authorizationKey,
       key: user.key,
     },
+    ...options,
```

Comment:
Just a note. This might overwrite the above core propery when present again on the options as we do not take the adapterArga apart carefully further down.

Reply:
True. Should we totally pass the `splitio()` config in the `adapterArgs` props?

Reply:
Let me try to propose the solution Ive got in my head to validate it latest by the weekend if that‘s cool. I‘d prefer playing with, thinking out solutions before having to publish the adapter with breaking changes twice. Hope thats ok.

Reply:
Yes no problem :)